package-url.md

```diff
@@ -0,0 +1,48 @@
+# purl format for Hex
+
+## Introduction
+
+[purl or Package URL](https://github.com/package-url/purl-spec) is a
+specification for identifying and locating software packages. It may be used
+to uniquely identify a package or package version in a software
+bill-of-materials (SBOM) or vulnerability report, or to locate the package in a
+repository, e.g. to check for updates.
+
+In particular, the [CycloneDX](https://cyclonedx.org) SBOM specification
+uses purl to identify components.
+
+This document formalizes the purl format for packages hosted on hex.pm or
+compatible repositories. Like Hex itself, it applies equally to packages
+written in/for Erlang, Elixir or any other BEAM language.
+
+## Definition
+
+* The 'type' is "hex"
+* The default repository is https://hex.pm
```

Comment:
```suggestion
* The default repository is https://repo.hex.pm
```

Reply:
Yeah, I guess, maybe... It does beg the question: can I reliably derive a URL that allows me to query the API, e.g. to look up the latest version of the package? Should I assume that the API always lives at the parent domain of the repo URL? Does the Hex standard (and the current implementation) have something to say about the relationship between the file repo and the API endpoints?

Reply:
No, the API and repository endpoints are independent. Maybe the specification have both `?repository_url` and `?api_url`?

Reply:
But the API URL cannot be determined by the tooling, can it? The only URL that is configured for a custom repo is the repository itself, because the Hex client only needs package artefacts and registries. In fact, I guess Hex servers other than hex.pm are probably not even required to expose an API

So yes, you are right, the repository_url default should be https://repo.hex.pm